arshad_khan/html/asignment7/professionaldetails.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <style>
+            div {
+                height: 600px;
+                width: 100%;
+            }
+            body {
+                background-image: url(/home/arshad/Downloads/Careers2.jpg);
+                background-repeat: no-repeat;
+                background-size: cover;
+            }
+            header {
+                width: 100%;
+                background-color: lightblue;
+                height: 50px;
+                text-align: left;
+            }
+            fieldset {
+                width: 500px;
+                height: 100%;
+            }
+        </style>
+    </head>
+    <body>
+        <div>
+            <header>
+                <center>
+                    <nav>
+                        <br />
+                        <a href="home.html">HOME |</a>
+                        <a href="newuserlogin.html">NEW USER LOGIN |</a>
+                        <a href="userdetails.html">USER DETAILS AND ACADEMIC DETAILS |</a>
+                        <a href="professionaldetails.html">PROFESSIONAL DETAILS</a>
+                    </nav>
+                </center>
+            </header>
+            <h2>This is Professionaldetails Page...</h2>
+            <center>
+                <fieldset>
+                    <legend>PROFESSIONALS DETAILS</legend>
+                    <form action="login">
+                        <label for="fname">First Name:</label>
+                        <label><input type="text" /></label>
```

Comment:
remove label tag
 <label><input type="text" /></label>

before and after input tag make this changes in all assesmment